sklearn/model_selection/tests/test_validation.py

```diff
@@ -713,6 +738,25 @@ def test_learning_curve_with_boolean_indices():
                               np.linspace(0.1, 1.0, 10))
 
 
+def test_learning_curve_with_shuffle():
+    X = np.array([[1, 2], [3, 4], [5, 6], [7, 8], [11, 12], [13, 14], [15, 16],
```

Comment:
Add a note here about why the test is designed this way, or perhaps just reference the issue number.